docs/api.rst

```diff
@@ -1,14 +1,40 @@
 API
 ===
 
-.. note::
-   On Unix / Linux, we follow the `XDG Basedir Spec`_. The spec allows overriding
-   directories with environment variables. The examples show are the default
-   values, alongside the name of the environment variable that overrides them.
 
-   See the spec itself for further details on the topic.
-
-.. _XDG Basedir Spec: https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html
+Platforms
```

Comment:
I'd list these classes last -- they're really not meant to be used by client applications

Client applications will usually want the platform-agnostic functions, since they'll want whatever is applicable in the current OS.

Reply:
Could swap them around 😊 No strong preference here.

Reply:
I'd keep the things we expect to be used most frequently first -- I assume those would be the platofrom-agnostic functions.

I can image the most popular use case being "gimme the config dir for this platform, whatever it may be".